Objects/perf_trampoline.c

```diff
@@ -185,6 +185,18 @@ struct trampoline_api_st {
 
 typedef struct trampoline_api_st trampoline_api_t;
 
+#if defined(__clang__) || defined(__GNUC__)
+extern void __clear_cache(void *, void*);
+#endif
+
+static void invalidate_icache(char* begin, char*end) {
```

Comment:
The call works on all architectures, is a no-op on x86-64 for example

Reply:
I tries the macro, but because this is a compiler Intrinsic autotools don't pick it up as if it was a function.

Or do you propose to define the macro manually?

Reply:
My comment is about the compiler warning:

> ‘invalidate_icache’ defined but not used [-Wunused-function]

On x86-64, your current PR defines the function but it's not used.

Reply:
Ah, gotcha!

Reply:
Done